### ISSUE TITLE
Move the category view to the index view

### ISSUE THREAD

**Issue by tyranja** [rubymonsters/speakerinnen_liste#509]
Right now we have am extra index view for the profiles ( which has a scoped view for topics and includes the search results ) and a view for the all profiles that belong to a category. It would be nice to have everything in one file.

**Comment by tyranja** [comment]
https://blog.heroku.com/solving-n-plus-one-queries

**Comment by tyranja** [comment]
I also changed the view of the speakerinnen:
Before: 
![old](https://cloud.githubusercontent.com/assets/1218914/24968579/f0df3706-1fad-11e7-956b-63955eb38679.png)


After: ( links and paragraph changed)
![new](https://cloud.githubusercontent.com/assets/1218914/24968470/9c3136aa-1fad-11e7-8996-8705c8eeec0c.png)

**Comment by tyranja** [comment]
Done
